fmralign/tests/test_utils.py

```diff
@@ -1,18 +1,33 @@
-from fmralign._utils import hierarchical_k_means, voxelwise_correlation
 import numpy as np
 from sklearn.utils.testing import assert_array_almost_equal
 from nilearn.input_data import NiftiMasker
 import nibabel
+import pytest
+from fmralign.tests.utils import random_niimg
+from fmralign._utils import _make_parcellation, voxelwise_correlation
 
 
-def test_hierarchical_k_means():
-    X = [[10, -10, 30], [12, -8, 24]]
-    truth_labels = np.tile([0, 1, 2], 5)
-    X = np.tile(X, 5).T
+def test_make_parcellation():
+    # make_parcellation is built on Nilearn which already has several test for its Parcellation class
+    # here we test just the call of the API is right on a simple example
+    img, mask_img = random_niimg((7, 6, 8, 5))
+    masker = NiftiMasker(mask_img=mask_img).fit()
+    n_pieces = 2
+    # create a predefined parcellation
+    labels_img = nibabel.Nifti1Image(
+        np.hstack([np.ones((7, 3, 8)), 2 * np.ones((7, 3, 8))]), np.eye(4))
+    for clustering_method in ["kmeans", "ward", labels_img]:
```

Comment:
maybe "rena" too ?

Reply:
For now "rena" is not released except for nilearn master version. I will test it as soon as it is in the newest release.